doc/toolbox-create.1.md

```diff
@@ -82,7 +82,8 @@ confusion.
 **--distro** DISTRO, **-d** DISTRO
 
 Create a toolbox container for a different operating system DISTRO than the
-host. Cannot be used with `--image`.
+host. Cannot be used with `--image`. Has to be coupled with `--release` unless
+the selected system matches the host system.
```

Comment:
Nitpick: just "...matches the host" seems enough, since we spelled out "operating system DISTRO ... host" before.